cmd/ci/main.go

```diff
@@ -24,53 +36,252 @@ type TestGroupMetadata struct {
+		return runners
+	}
+
+	return []Runner{linux_x64}
+}
+
+func getTestGroupMetadata(testGroupName string) TestGroupMetadata {
+	packages := getPackages(testGroupName)
+	needsParameters := getNeedsParameters(testGroupName)
+	skipConformance := getSkipConformance(testGroupName)
+	testRustProofLogs := getTestRustProofLogs(testGroupName)
+	format := getFormat()
+	goTestFlags := getGoTestFlags(testGroupName)
+
+	return TestGroupMetadata{
+		Packages:          packages,
+		NeedsParameters:   needsParameters,
+		SkipConformance:   skipConformance,
+		TestRustProofLogs: testRustProofLogs,
+		Format:            format,
+		GoTestFlags:       goTestFlags,
+	}
+}
+
+func getPackages(testGroupName string) []string {
+	testGroupNamesToPackages := map[string][]string{
+		"multicore-sdr": {createPackagePath("storage", "sealer", "ffiwrapper")},
+		"conformance":   {createPackagePath("conformance")},
 		"unit-cli": {
-			strings.Join([]string{".", "cli", "..."}, string(os.PathSeparator)),
-			strings.Join([]string{".", "cmd", "..."}, string(os.PathSeparator)),
-			strings.Join([]string{".", "api", "..."}, string(os.PathSeparator)),
+			createPackagePath("cli", "..."),
+			createPackagePath("cmd", "..."),
+			createPackagePath("api", "..."),
 		},
 		"unit-storage": {
-			strings.Join([]string{".", "storage", "..."}, string(os.PathSeparator)),
-			strings.Join([]string{".", "extern", "..."}, string(os.PathSeparator)),
+			createPackagePath("storage", "..."),
+			createPackagePath("extern", "..."),
 		},
 		"unit-node": {
-			strings.Join([]string{".", "node", "..."}, string(os.PathSeparator)),
+			createPackagePath("node", "..."),
 		},
 		"unit-rest": {
-			strings.Join([]string{".", "blockstore", "..."}, string(os.PathSeparator)),
-			strings.Join([]string{".", "build", "..."}, string(os.PathSeparator)),
-			strings.Join([]string{".", "chain", "..."}, string(os.PathSeparator)),
-			strings.Join([]string{".", "conformance", "..."}, string(os.PathSeparator)),
-			strings.Join([]string{".", "gateway", "..."}, string(os.PathSeparator)),
-			strings.Join([]string{".", "journal", "..."}, string(os.PathSeparator)),
-			strings.Join([]string{".", "lib", "..."}, string(os.PathSeparator)),
-			strings.Join([]string{".", "paychmgr", "..."}, string(os.PathSeparator)),
-			strings.Join([]string{".", "tools", "..."}, string(os.PathSeparator)),
+			createPackagePath("blockstore", "..."),
+			createPackagePath("build", "..."),
+			createPackagePath("chain", "..."),
+			createPackagePath("conformance", "..."),
+			createPackagePath("gateway", "..."),
+			createPackagePath("journal", "..."),
+			createPackagePath("lib", "..."),
+			createPackagePath("paychmgr", "..."),
+			createPackagePath("tools", "..."),
 		},
 	}
 
-	if strings.HasPrefix(name, "itest-") {
-		return []string{strings.Join([]string{".", "itests", strings.Join([]string{strings.TrimPrefix(name, "itest-"), "test.go"}, "_")}, string(os.PathSeparator))}
+	if strings.HasPrefix(testGroupName, "itest-") {
```

Comment:
(nit) but maybe move this if to the top of the function to avoid the interleaving of concerns that we have now

nonItest map declaration
itest handling
nonItest handling